File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_latlong/Place.java
package com.android.blawniczak.astroweather.domain.woeid_latlong;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Place {

    @SerializedName("centroid")
    @Expose
    public Centroid centroid;

    @SerializedName("woeid")
    @Expose
    public String woeid;

}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_latlong/Place.kt
package com.android.blawniczak.astroweather.domain.woeid_latlong

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Place {

    @SerializedName("centroid")
    @Expose
    var centroid: Centroid? = null

    @SerializedName("woeid")
    @Expose
    var woeid: String? = null

}
